### PR TITLE
wire print flags through apply cmd

### PR DESCRIPTION
**Pull request by juanvallejo**
**Release note**:
```release-note
NONE
```

Depends on https://github.com/kubernetes/kubernetes/pull/62300
Adds PrintFlags to `apply` command.

cc @soltysh @deads2k

### CODE REVIEW

File: pkg/kubectl/cmd/apply.go [hunk 1]
```diff
@@ -159,6 +174,26 @@ func NewCmdApply(baseName string, f cmdutil.Factory, out, errOut io.Writer) *cob
 	return cmd
 }
 
+func (o *ApplyOptions) Complete(cmd *cobra.Command) error {
+	o.DryRun = cmdutil.GetDryRunFlag(cmd)
+
+	// allow for a success message operation to be specified at print time
+	o.PrintObj = func(obj runtime.Object, operation string) error {
```
Comment (deads2k) [inline pkg/kubectl/cmd/apply.go]: Icky, but better than we have I suppose.

`func(operation string) printers.ResourcePrinterFunc` though to make it clear what is happening.

**Review by soltysh — APPROVED** [review]
/approve no-issue
Rebase and I'll apply the lgtm label

**Comment by juanvallejo** [comment]
@soltysh thanks, rebased

**Comment by juanvallejo** [comment]
Added a test

**Comment by soltysh** [comment]
> @soltysh thanks, rebased

I know, but I was too slow :sob:

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**